benchmarking/README.md

```diff
@@ -17,3 +17,24 @@ python -m imitation.scripts.<train_script> <algo> with benchmarking/<config_name
 ...
 ex.add_config('benchmarking/<config_name>.json')
 ```
+
+# Tuning Hyperparameters
+
+The hyperparameters of any algorithm in imitation can be tuned using the `tuning.py` script.
+The benchmarking hyperparameter configs were generated by tuning the hyperparameters using
+the search space defined in the `tuning_config.py` script. The tuning script proceeds in two
+phases: 1) The hyperparameters are tuned using the search space provided, and 2) the best 
+hyperparameter config found in the first phase based on the maximum mean return is
+re-evaluated on a separate set of seeds, and the mean and standard deviation of these trials
+are reported.
+
+To tune the hyperparameters of an algorithm using the default search space provided:
+```bash
+python tuning.py with example_{algo} 'parallel_run_config.base_named_configs=["{env}"]'
```

Comment:
Why is there the `example_` prefix?

Reply:
A vestigial naming convention left from the earlier code of parallel. Have thought about removing it for quite some time. Will remove it now.